api/opencl_runtime_layer.asciidoc

```diff
@@ -1210,57 +1207,52 @@ then be reused by the application.
 if the function is executed successfully.
 Otherwise, they return one of the following errors:
 
-  * {CL_INVALID_COMMAND_QUEUE} if _command_queue_ is not a valid host
-    command-queue.
-  * {CL_INVALID_CONTEXT} if the context associated with _command_queue_ and
-    _buffer_ are not the same or if the context associated with
-    _command_queue_ and events in _event_wait_list_ are not the same.
-  * {CL_INVALID_MEM_OBJECT} if _buffer_ is not a valid buffer object.
-  * {CL_INVALID_VALUE} if _buffer_origin_, _host_origin_, or _region_ is `NULL`.
-  * {CL_INVALID_VALUE} if the region being read or written specified by
-    (_buffer_origin_, _region_, _buffer_row_pitch_, _buffer_slice_pitch_) is
-    out of bounds.
-  * {CL_INVALID_VALUE} if any _region_ array element is 0.
-  * {CL_INVALID_VALUE} if _buffer_row_pitch_ is not 0 and is less than
-    _region_[0].
-  * {CL_INVALID_VALUE} if _host_row_pitch_ is not 0 and is less than
-    _region_[0].
-  * {CL_INVALID_VALUE} if _buffer_slice_pitch_ is not 0 and is less than
-    _region_[1] {times} _buffer_row_pitch_ and not a multiple of
-    _buffer_row_pitch_.
-  * {CL_INVALID_VALUE} if _host_slice_pitch_ is not 0 and is less than
-    _region_[1] {times} _host_row_pitch_ and not a multiple of
-    _host_row_pitch_.
-  * {CL_INVALID_VALUE} if _ptr_ is `NULL`.
-  * {CL_INVALID_EVENT_WAIT_LIST} if _event_wait_list_ is `NULL` and
-    _num_events_in_wait_list_ > 0, or _event_wait_list_ is not `NULL` and
-    _num_events_in_wait_list_ is 0, or if event objects in _event_wait_list_
-    are not valid events.
-  * {CL_MISALIGNED_SUB_BUFFER_OFFSET} if _buffer_ is a sub-buffer object and
-    _offset_ specified when the sub-buffer object is created is not aligned
-    to {CL_DEVICE_MEM_BASE_ADDR_ALIGN} value for device associated with
-    _queue_.
+  * {CL_INVALID_COMMAND_QUEUE}
+    ** if _command_queue_ is not a valid host command-queue
+  * {CL_INVALID_CONTEXT}
+    ** if the context associated with _command_queue_ and _buffer_ are not the same
+    ** if the context associated with _command_queue_ and events in _event_wait_list_ are not the same
+  * {CL_INVALID_MEM_OBJECT}
+    ** if _buffer_ is not a valid buffer object
+  * {CL_INVALID_VALUE}
+    ** if _buffer_origin_, _host_origin_, or _region_ is `NULL`
+    ** if the region being read or written specified by _buffer_origin_, _region_, _buffer_row_pitch_, and _buffer_slice_pitch_ is out of bounds
+    ** if any _region_ array element is zero
+    ** if _buffer_row_pitch_ is not equal to zero and is less than _region_[0]
+    ** if _buffer_slice_pitch_ is not equal to zero and is less than the_region_[1] times _buffer_row_pitch_
```

Comment:
```suggestion
    ** if _buffer_slice_pitch_ is not equal to zero and is less than _region_[1] times _buffer_row_pitch_
```

there was also `{times}` rather than `times` before, not sure if that was intentional.

Reply:
Yeah, the change from `{times}` to `times` was intentional.

We have a number of different "conventions" throughout the spec, ranging from `*` to `X` to `{times}` (which expands to something like `X`) to the text `times`.  I think text descriptions like `times` is the most descriptive, so I've been trying to switch to it more consistently, though I'm sure this will be an ongoing process.